code/edgemedia/src/main/java/com/adobe/marketing/mobile/edge/media/internal/MediaCollectionConstants.java

```diff
@@ -16,7 +16,7 @@ class MediaCollectionConstants {
     private MediaCollectionConstants() {}
 
     static final String MEDIA_LIBRARY_VERSION =
-            "android-media-" + com.adobe.marketing.mobile.Media.extensionVersion();
+            "android-media-" + com.adobe.marketing.mobile.edge.media.Media.extensionVersion();
```

Comment:
We can remove this file entirely.

Reply:
I'll leave that for future work in another PR when we add the MediaSession and MediaEventTracker classes.